Leanplum-SDK/Classes/Managers/Networking/LPRequestManager.m

```diff
@@ -389,7 +393,18 @@ - (void)sendRequests:(BOOL)async
     }
 }
 
--(void)uiDidTimeout {
+- (void)sendAllCounts {
+    NSDictionary *counts = [[LPCountManager sharedManager] getAndClearCounts];
+    for (id key in counts) { // iterate over counts, creating one request per counter
```

Comment:
Let's not use bare `id` s unless necessary. We can make this `NSString *key` instead